src/Components/VideoActions/VideoActions.jsx

```diff
@@ -2,13 +2,167 @@ import React, { useEffect, useState } from "react";
 import "./VideoActions.css";
 import { useAuth } from "../../Context/Auth-Context";
 import { useUserData } from "../../Context/UserData-Context";
+import toast, { Toaster } from "react-hot-toast";
 
 import axios from "axios";
 const VideoActions = (props) => {
-  
+  const {
```

Comment:
Suggestion: You can extract the whole logic to a separate Container named VideoActionsContainer and use it like a custom hook in the view(VideoActions) component.